src/foam/doc/DigDeleteView.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.doc',
+  name: 'DigDeleteView',
+  extends: 'foam.u2.View',
+  documentation: 'View used to show snippets of deletion API calls.',
+  implements: [
+    'foam.mlang.Expressions',
```

Comment:
I don't think you need this implements. This adds functions for creating expressions easily but I don't think you're using any here.

Reply:
@mcarcaso .. interesting. how can you see either the functionality added or the source class that adds these methods?

Reply:
found it, thanks mike .. i've remove it from the views and everything works as expected.